pages/gallery.html

```diff
@@ -14,16 +14,52 @@
     <title>Galeria - Hostal las paisas</title>
   </head>
   <body>
-    <nav>
-      <!-- nav bar section -->
-    </nav>
     <section class="small-hero small-hero__image--gallery ">
       <div class="small-hero__layer"></div>
       <div class="small-hero__layer small-hero__layer--darker"></div>
       <h1 class="small-hero__title">Galería</h1>
     </section>
+    <nav>
+      <!-- nav bar section -->
+    </nav>
+    <!-- gallery -->
+    <section class="container containes-componet">
+        <div class="info-gallery__text">    
+            <p class="text-p">Nuestro hostal te ofrece todas las comodidades para que vivas la mejor experiencia. Observa todo lo que tenemos para ti en el majestuoso paraíso de Capurganá</p>
+        </div>
+            <div>
+                <div class="container-images" id="divImagesContainer" >
+                    <div class="container-images__item"><div class="item__images"></div>
+                    <div class="item__overlay"><img src="./../assets/images/gallery/expand_icon.svg" class="item__overlay--image" alt=""></div>
+                    </div>
+                    <div class="container-images__item"><div class="item__images"></div>
+                    <div class="item__overlay"><img src="./../assets/images/gallery/expand_icon.svg" class="item__overlay--image" alt=""></div>
+                    </div>
+                    <div class="container-images__item"><div class="item__images"></div>
+                    <div class="item__overlay"><img src="./../assets/images/gallery/expand_icon.svg" class="item__overlay--image" alt=""></div>
+                    </div>
+                    <div class="container-images__item"><div class="item__images"></div>
+                    <div class="item__overlay"><img src="./../assets/images/gallery/expand_icon.svg" class="item__overlay--image" alt=""></div>
+                    </div>
+                    <div class="container-images__item"><div class="item__images"></div>
+                    <div class="item__overlay"><img src="./../assets/images/gallery/expand_icon.svg" class="item__overlay--image" alt=""></div>
+                    </div>
+                    <div class="container-images__item"><div class="item__images"></div>
+                    <div class="item__overlay"><img src="./../assets/images/gallery/expand_icon.svg" class="item__overlay--image" alt=""></div>
+                    </div>
+                    <div class="container-images__item"><div class="item__images"></div>
+                    <div class="item__overlay"><img src="./../assets/images/gallery/expand_icon.svg" class="item__overlay--image" alt=""></div>
+                    </div>
+                    <div class="container-images__item"><div class="item__images"></div>
+                    <div class="item__overlay"><img src="./../assets/images/gallery/expand_icon.svg" class="item__overlay--image" alt=""></div>
```

Comment:
why do you use `.item__overlay` on the parent and `.item__overlay--image` on the child if both set a display none?

Reply:
because one is to hide the black image that goes on the image and the other is to control the size of the image

Reply:
im referring to `gallery.css` lines 104 and 107

Reply:
when used in movile it should not have the effect

Reply:
`item__overlay` contiene a `item__overlay--image`, por tanto si `item__overlay`tiene display none, sus hijos ya estan ocultos, no hace falta ponerles display none tambien.

otra cosa `item__overlay--image` creo que deberia ser `overlay__image` ya que image es un elemnto y no un modificador.